components/style/gecko/rules.rs

```diff
@@ -67,6 +68,16 @@ impl ToNsCssValue for font_feature_settings::T {
     }
 }
 
+impl ToNsCssValue for font_language_override::SpecifiedValue {
+    fn convert(self, nscssvalue: &mut nsCSSValue) {
+        match self {
+            font_language_override::SpecifiedValue::Normal => nscssvalue.set_normal(),
+            font_language_override::SpecifiedValue::Override(ref lang) => nscssvalue.set_string(&*lang),
+            font_language_override::SpecifiedValue::System(_) => {},
```

Comment:
The `System`, as the name implies, means it's set by the system. So we don't have a parse mechanism for that. It's set during the conversion to ComputedValue. And also to be able to get that system font, we need to be able to get `Context` which we don't have in this method.